newdle/client/src/components/MyNewdles.js

```diff
@@ -1,24 +1,45 @@
 import React from 'react';
-import {Container, Icon, Label, Placeholder} from 'semantic-ui-react';
+import {Button, Container, Icon, Label, Placeholder} from 'semantic-ui-react';
 import {useHistory} from 'react-router';
 import {serializeDate, toMoment} from '../util/date';
 import client from '../client';
 import styles from './MyNewdles.module.scss';
 
 export default function MyNewdles() {
   const [newdles] = client.useBackend(() => client.getMyNewdles(), []);
+  const history = useHistory();
 
   return (
     <Container text className={styles.newdles}>
-      {newdles === null ? (
+      {newdles === null && (
         <>
           <Placeholder className={styles.newdle} />
           <Placeholder className={styles.newdle} />
           <Placeholder className={styles.newdle} />
         </>
-      ) : (
-        newdles.map(newdle => <MyNewdle key={newdle.id} newdle={newdle} />)
       )}
+      {newdles !== null && newdles.length === 0 && (
+        <div className={styles['no-newdle-container']}>
+          <h2>You didn't create any Newdles yet...</h2>
+          <div>
+            <Button
+              color="violet"
+              type="submit"
+              onClick={() => {
+                history.push('/new');
+              }}
+            >
+              Create your First Newdle!{' '}
+              <span role="img" aria-label="Newdle">
```

Comment:
I don't think we need aria roles here since this icon is just to make the button prettier - if someone cannot see icons (e.g. in a screenreader), they do not miss any important information by not having it.

And it would probably be confusing if it was read as "Create your first newdle. Newdle." or something similar...